README.md

```diff
@@ -42,9 +43,6 @@ You can also run the application directly on the host machine by defining enviro
     LIBCAL_CLIENT2_SECRET=0123456789abcdef0123456789abcdef \
     LIBCAL_TOKEN_ENDPOINT=https://calendar.library.ucla.edu/1.1/oauth/token \
     LIBCAL_BASE_URL=https://calendar.library.ucla.edu \
-    LIBCAL_AUTH_RETRY_COUNT=3 \
-    LIBCAL_AUTH_RETRY_DELAY=10 \
-    LIBCAL_AUTH_EXPIRES_IN_PADDING=300 \
```

Comment:
Even if we eventually change how the app is configured in this regard (which I am on board with), at this point these values are still configurable, so I don't see why this should be removed yet. For example maybe a dev wants to run a local instance with these values tweaked in order to manually test the authentication process (which I've done in the past).

Reply:
Maybe mentioning them in the text below as optional arguments would be better? The way that it is currently makes them look required.